src/requests/payment-request.ts

```diff
@@ -85,7 +85,12 @@ export async function createPaymentRequest(
   // payment requests can only be created for flows with a pay component
   // this throws if the pay component cannot be found
   const payNode = await getPayNode(client, session);
-  const govPayMetadata = payNode.data.govPayMetadata;
+  const govPayMetadata = payNode.data?.govPayMetadata || [
+    { key: "source", value: "PlanX" },
+    { key: "inviteToPay", value: true },
```

Comment:
nit: should the default value for `inviteToPay` be `false` since we know it's seldom used currently ?

Reply:
Good question! In this instance, `true` is correct as this will be the value used for a `payment_request` record as they're used for invite to pay journeys only.